.github/workflows/central-config-tests.yaml

```
@@ -0,0 +1,79 @@
+        uses: konveyor/tackle2-operator/.github/actions/start-minikube@main
+        with:
+          memory: "max"
+          cpus: "max"
+
+      - name: Install Konveyor
+        uses: konveyor/tackle2-operator/.github/actions/install-konveyor@main
+        with:
+          operator-bundle-image: quay.io/konveyor/tackle2-operator-bundle:${{ github.event.inputs.operator_tag || 'latest' }}
+
+      - name: Wait for Hub and get URL
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:3:28: Double quote to prevent globbing and word splitting

(shellcheck)